### PR TITLE
[OAP-1811][OAP-cache][POAE7-486]add sbin folder

### PR DESCRIPTION
**Pull request by offthewall123**
## What changes were proposed in this pull request?

1:start/install_plasma_server.sh
2:start/install_redis_server.sh

### CODE REVIEW

**Comment by offthewall123** [comment]
@jikunshang @yma11 @winningsix @yeyuqiang Please take a review ,thanks.

**Comment by jikunshang** [comment]
#1811

File: sbin/start_plasma_server.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# install arrow and plasma
```
Comment (jikunshang) [inline sbin/start_plasma_server.sh:1]: this file should rename to install_plasma.sh?

Comment (jikunshang) [reply sbin/start_plasma_server.sh:1]: add "#! /bin/bash"

File: sbin/start_plasma_server.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+cd release
+
+# build libarrow, libplasma, libplasma_java
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_FLAGS="-g -O3" -DCMAKE_CXX_FLAGS="-g -O3" -DARROW_BUILD_TESTS=on -DARROW_PLASMA_JAVA_CLIENT=on -DARROW_PLASMA=on -DARROW_DEPENDENCY_SOURCE=BUNDLED ..
+make -j40
+sudo make install -j40
+
+# copy libs to lib path
+sudo rm /lib64/libarrow.so*
+sudo cp $DIR/cpp/release/release/libarrow.so.100.0.0 /lib64/
```
Comment (jikunshang) [inline sbin/start_plasma_server.sh:19]: version?

**Review by jikunshang — COMMENTED** [review]
I think plasma is recommended to install by conda

File: sbin/start_redis_server.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# install redis
+mkdir redisInstall
+cd redisInstall
+wget https://download.redis.io/releases/redis-6.0.8.tar.gz
+tar xzf redis-6.0.8.tar.gz
+cd redis-6.0.8
+make
+
+./src/redis-server --protected-mode no
```
Comment (jikunshang) [inline sbin/start_redis_server.sh:9]: don't need to install?  Install and start service should separate into two script?

Comment (offthewall123) [reply sbin/start_redis_server.sh:9]: Start service is just one line, no need to separate this?

Comment (jikunshang) [reply sbin/start_redis_server.sh:9]: if a user want to start a redis server, this script will build source code every time.

File: sbin/start_plasma_server.sh (hunk 1 above)
Comment (offthewall123) [reply sbin/start_plasma_server.sh]: Done

File: sbin/start_redis_server.sh (hunk 3 above)
Comment (offthewall123) [reply sbin/start_redis_server.sh:2]: Got it

File: sbin/start_redis_server.sh [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./src/redis-server --protected-mode no
```
Comment (jikunshang) [inline sbin/start_redis_server.sh:2]: need some configuration?

Comment (offthewall123) [reply sbin/start_redis_server.sh:2]: No more additional configuration when starting it....

Comment (jikunshang) [reply sbin/start_redis_server.sh:2]: Add some notice like "Please change Redis setting in xxs file according to your environment."

Comment (offthewall123) [reply sbin/start_redis_server.sh:8]: Done

**Pull request merged**